Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2274,6 +2282,100 @@ - (void)writingToolsSession:(WTSession *)session didReceiveAction:(WTAction)acti
     _page->writingToolsSessionDidReceiveAction(*webSession, webAction);
 }
 
+#pragma mark - WKIntelligenceTextEffectCoordinatorDelegate conformance
+
+// FIXME: Give the `coordinator` parameters actual types.
+
+- (PlatformView *)viewForIntelligenceTextEffectCoordinator:(id /* WKIntelligenceTextEffectCoordinator * */)coordinator
+{
+    return self;
+}
+
+- (void)intelligenceTextEffectCoordinator:(id /* WKIntelligenceTextEffectCoordinator * */)coordinator rectsForProofreadingSuggestionsInRange:(NSRange)range completion:(void (^)(NSArray<NSValue * /* CGRect/NSRect */> *))completion
```

Comment:
Instead of `NSArray<NSValue * /* CGRect/NSRect */> *`, it might be clearer if you just made it `NSArray<NSValue *> *` (since the `rectsForProofreadingSuggestionsInRange:` part of the selector already implies that these are rect values). Alternately, you could consider using a `typedef` for the completion handler type.